### PR TITLE
{bio} Gromacs for CrayIntel 2015.11-XC

### PR DESCRIPTION
**Pull request by gppezzi**

### CODE REVIEW

**Comment by gppezzi** [comment]
Test report by @gppezzi
**FAILED**
Build succeeded for 4 out of 5 (3 easyconfigs in this PR)
dora103 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/bcb86172c27cd573fb1b for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
dora103 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/c309eb778e7a52198394 for a full test report.

**Comment by gppezzi** [comment]
It failed the first time because my metadata file was not in sync for fftw (now we use the relative path).

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
swan - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/ff1a9594ad4eb7dcefea for a full test report.

**Comment by gppezzi** [comment]
@boegel ping? this is basically the same as CrayGNU version (try toolchain).

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name = 'ncurses'
+version = '5.9'
+
+homepage = 'http://www.gnu.org/software/ncurses/'
+description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
+ and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
+ function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2015.11-XC'}
+toolchainopts = {'optarch': True, 'shared': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: `shared` is enabled by default, so is `optarch`?

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['ncurses-%(version)s_configure_darwin.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: a bit weird to have this here (this patch is specific to OS X)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
daint104 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/4049a2add0529c23c427 for a full test report.

**Comment by boegel** [comment]
@gppezzi: any idea why this doesn't work on Daint?

```
-- checking for module 'fftw3f'
--   package 'fftw3f' not found
-- pkg-config could not detect fftw3f, trying generic detection
Could not find fftw3f library named libfftw3f, please specify its location in CMAKE_PREFIX_PATH or FFTWF_LIBRARY by hand (e.g. -DFFTWF_LIBRARY='/path/to/libfftw3f.so')
CMake Error at CMakeLists.txt:928 (MESSAGE):
  Cannot find FFTW 3 (with correct precision - libfftw3f for single-precision
  GROMACS or libfftw3 for double-precision GROMACS).  Either choose the right
  precision, choose another FFT(W) library (-DGMX_FFT_LIBRARY), enable the
  advanced option to let GROMACS build FFTW 3 for you
  (-DGMX_BUILD_OWN_FFTW=ON) , or use the really slow GROMACS built-in fftpack
  library (-DGMX_FFT_LIBRARY=fftpack).
```

**Comment by boegel** [comment]
@gppezzi ah, hold on, some issue as you ran into, old metadata... Testing again.

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb (hunk 2 above)
Comment (gppezzi) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: also here I suppose https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayGNU-2015.11-XC.eb#L17

Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: yes, sure, remove it there too, doesn't make sense

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
daint104 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/5b31e1b94b1ee71645d3 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
dora104 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/5be4cc4b462f97720096 for a full test report.

**Comment by gppezzi** [comment]
@boegel fixed remarks (ncurses)

**Comment by boegel** [comment]
OK, going in. Thanks @gppezzi !

**Pull request merged**